src/lib/components/wizard-navigation-bar.component.horizontal.less

```diff
@@ -47,7 +47,6 @@
   border-radius: 100%;
 }
 .state-circle-hover(@dot-width, @dot-height, @dot-border-width) { }
-.state-circle-nohover(@dot-width, @dot-height, @dot-border-width) { }
```

Comment:
Just for my understanding:
You removed the `nohover` functions.
How did you provide the `nohover` behavior afterwards?

Reply:
Simple -- `hover` state is now only applied to `.navigable` items. I did not find any issues with this during testing.